cfgov/unprocessed/js/organisms/Header.js

```diff
@@ -53,10 +53,13 @@ function Header( element ) {
     _globalSearch.addEventListener( 'expandBegin', _searchExpandBegin );
     _globalSearch.init();
 
-    _megaMenu = new MegaMenu( _dom );
-    _megaMenu.addEventListener( 'rootExpandBegin', _megaMenuExpandBegin );
-    _megaMenu.addEventListener( 'rootCollapseEnd', _megaMenuCollapseEnd );
-    _megaMenu.init();
+    // Don't initialize the mega menu if it isn't on the page.
+    if ( _dom.classList.contains( 'o-header__mega-menu' ) ) {
```

Comment:
Update to ``if ( _dom.classList.contains( `${ BASE_CLASS }__mega-menu` ) ) {``

Reply:
Fixed in 866cd10.